### PR TITLE
Additional color refinement

### PR DESCRIPTION
**Pull request by aaronshekey**
This spins up a color documentation page while incorporating feedback from the larger design team. Yellow gets a little more orange. Greens get a little more saturated.

### CODE REVIEW

File: .storybook/main.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ const config: StorybookConfig = {
 			to: "/static",
 		},
 	],
-	stories: [relativePath("..", "src", "**", "*.stories.@(js|jsx|ts|tsx|mdx)")],
+	stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
```
Comment (cody-dot-js) [inline .storybook/main.ts:47]: you _don't_ want the `mdx` stories to match?

Comment (aaronshekey) [reply .storybook/main.ts:47]: There's a tendency in the Storybook examples that the straight up docs pages don't include the `.stories` naming. Or something.

Comment (aaronshekey) [reply .storybook/main.ts]: Also copypasta.

File: .storybook/main.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ const config: StorybookConfig = {
 			to: "/static",
 		},
 	],
-	stories: [relativePath("..", "src", "**", "*.stories.@(js|jsx|ts|tsx|mdx)")],
+	stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx|mdx)"],
```
Comment (cody-dot-js) [inline .storybook/main.ts:47]: just be aware that this _could_ break cross compat w/ other OSes by not using path.join 😞

**Pull request merged**